invoke.yaml

```diff
@@ -52,8 +52,16 @@ osm: # A file or a url MUST be defined
 #     nb_replicas:  # control the number of replicas of the ES index
 
 addresses:
+  # Use the deduplication to merge addresses from different sources.
+  #
+  # NOTE: If this is false, only addresses from OpenAddresses may be imported.
```

Comment:
This comment is not correct. With the deduplication disabled, one of BANO or OA may be used.